### PR TITLE
Package ocsigen-i18n.4.0.0

### PR DESCRIPTION
**Pull request by jrochel**
### `ocsigen-i18n.4.0.0`
I18n made easy for web sites written with eliom
Provides executables: ocsigen-i18n-generator for generating an eliom file from a file containing tab-separated values; ocsigen-i18n-rewriter for implementing a PPX syntax for referencing entries in the generated eliom file

---
* Homepage: https://github.com/besport/ocsigen-i18n
* Source repo: git+https://github.com/besport/ocsigen-i18n.git
* Bug tracker: https://github.com/besport/ocsigen-i18n/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ocsigen-i18n/ocsigen-i18n.4.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis:     "I18n made easy for web sites written with eliom"
+description:  "Provides executables: ocsigen-i18n-generator for generating an eliom file from a file containing tab-separated values; ocsigen-i18n-rewriter for implementing a PPX syntax for referencing entries in the generated eliom file"
+maintainer:   "Jan Rochel <jan@besport.com>"
+
+homepage: "https://github.com/besport/ocsigen-i18n"
+bug-reports: "https://github.com/besport/ocsigen-i18n/issues"
+dev-repo: "git+https://github.com/besport/ocsigen-i18n.git"
+build:   [ "dune" "build" "-p" name "-j" jobs ]
+
```
Comment (kit-ty-kate) [inline packages/ocsigen-i18n/ocsigen-i18n.4.0.0/opam:10]:
```suggestion
license: "??"
```
The license is missing, could you add it?

File: packages/ocsigen-i18n/ocsigen-i18n.4.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.7"}
+  "ppxlib" {> "0.20"}
```
Comment (kit-ty-kate) [inline packages/ocsigen-i18n/ocsigen-i18n.4.0.0/opam:14]:
```suggestion
  "ppxlib" {>= "0.21.0"}
```
This package requires 0.21.0 or later. (0.20.0 being > 0.20):
```
# File "i18n_ppx_checker.ml", line 66, characters 15-47:
# 66 |               (Expansion_context.Base.top_level
#                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: This function has type
#          tool_name:string ->
#          file_path:string -> Ppxlib.Expansion_context.Base.t
#        It is applied to too many arguments; maybe you forgot a `;'.



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+- The following actions failed
| - build ocsigen-i18n 4.0.0
```

**Comment by jrochel** [comment]
@kit-ty-kate Oops, I have force pushed a version that has overwritten your changes! I'd like to restore them but they seem to have been garbage collected on github. Could you force push your version again?

**Comment by kit-ty-kate** [comment]
@jrochel done

**Comment by jrochel** [comment]
Thanks for all your work on this, btw!

**Comment by jrochel** [comment]
@kit-ty-kate Is there anything I could/should do about this PR? I have committed your changes to the opam file to the ocsigen-i18n repository.

**Comment by kit-ty-kate** [comment]
It looks fine as is. Thanks!

**Pull request merged**